reana/cli.py

```diff
@@ -375,6 +378,8 @@ def select_components(components):
                                 cover all REANA client components;
                           * (6) special value 'ALL' that will expand to include
                                 all REANA repositories.
+                          * (7) special value 'ALL-EXAMPLES' that will exapand
```

Comment:
Perhaps the special value could be just `EXAMPLES`, or `DEMO`, again to have nice analogy with already-existing  `CLUSTER` (and `ALL`) special values... Using again the optics of one single word without dashes.  If `CLUSTER` denotes all repositories concerned with cluster builfing, then `DEMO` could denote all `reana-demo...` examples repositories.  That kind of naming logic.

Reply:
I would go for `DEMO` then, I am amending this and previous comments, thanks!